src/bin/todo/update.rs

```diff
@@ -1,8 +1,39 @@
+use chrono::NaiveDateTime;
 use clap::Parser;
+use mindmap::Priority;
 
 #[derive(Parser)]
-pub struct Args {}
+pub struct Args {
+    #[clap(long)]
+    description: String,
+    #[clap(long)]
+    new_description: String,
+    #[clap(long)]
+    new_priority: Priority,
+    #[clap(long, value_parser=mindmap::parse_datetime)]
+    new_deadline: NaiveDateTime,
+}
+
+pub async fn command(_args: &Args) {
```

Comment:
The underscore in `_args` specifies that it's an unused variable. If we're using it, remove the underscore.

Reply:
Oh, okay